.github/workflows/deploy.yml

```
@@ -0,0 +1,42 @@
+  # BUILD
+  build:
+    name: Build Site
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Pages
+        uses: actions/configure-pages@v5
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.13' 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)